runners/katacoda/templates/intro.md

```diff
@@ -1,8 +1,7 @@
 <%= description; %>
 
 <% if(tutorialPath){ %>
-The definition of each step of this tutorial can be found at https://github.com/devonfw-tutorials/tutorials/tree/main/<%= tutorialPath; %>
+The definition of each step of this tutorial can be found at https://github.com/devonfw-tutorials/tutorials/tree/main/<%= tutorialPath; %>. <% } %>
 
-Feel free to fix any errors you find yourself. You just need to create a pull request to the tutorials repository with your changes.
-You can find a description of what to look for when creating a pull request at the devonfw contribution guide: https://devonfw.com/website/pages/docs/CONTRIBUTING.asciidoc.html#contributing.asciidoc_pull-requests
-<% } %>
\ No newline at end of file
+Feel free to report any errors to us or fix them yourself. You just need to create a pull request to the https://github.com/devonfw-tutorials/tutorials[tutorials repository] with your changes.
```

Comment:
In the first sentence you could mention that the errors can be reported by creating an issue in the tutorials repo. Also in the wiki template